docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -0,0 +1,599 @@
+
+Dynamic Zone fields let you create a flexible space in which to compose content, based on a mixed list of components.
+
+They accept a `components` array that follows this format: `<category>.<componentName>`.
+
+```json
+// ./api/[api-name]/content-types/article/schema.json
+
+{
+  "attributes": {
+    "body": {
+      "type": "dynamiczone",
+      "components": ["article.slider", "article.content"]
+    }
+  }
+}
+```
+
+### Model options
+
+The `options` key on the in the model description can use the following keys:
+
+| Key                     | Type                        | Description                                                                                                                                                                                                                                                                                                                                  |
+| ----------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `privateAttributes`     | Array of strings            | This configuration allows to treat a set of attributes as private, even if they're not actually defined as attributes in the model. It could be used to remove from API responses timestamps.<br><br>The set of `privateAttributes` defined in the model are merged with the `privateAttributes` defined in the global Strapi configuration. |
+| `populateCreatorFields` | Boolean                     | Configure whether the API response should include `created_by` and `updated_by` fields or not.<br><br>Default value: `false`                                                                                                                                                                                                                 |
+| `draftAndPublish`       | Boolean                     | Enable the draft and publish feature.<br><br>Default value: `false`                                                                                                                                                                                                                                                                          |
+
+
+```json
+// .api/[api-name]/content-types/restaurant/schema.json
+
+{
+  "options": {
+    "timestamps": true,
+    "privateAttributes": ["id", "created_at"],
+    "populateCreatorFields": true,
+    "draftAndPublish": false
+  }
+}
+```
+
+## Lifecycle hooks
+
+The lifecycle hooks are functions that get triggered when the Strapi queries are called. They are triggered automatically when you manage your content in the Admin Panel or when you develop custom code using `queries`·
```

Comment:
May be useful to warn that they are not called in case you use directly the `knex` library instead of strapi functions.